### ISSUE TITLE
Individual Reflection

### ISSUE THREAD

**Issue by wenshilanR** [JoyfulTechLauncher/joyfulfrugalista#40]
From shadow team: The team is documenting feedback on GitHub, using it as a foundation for reflective discussions and strategy adjustments. They also focus on continuous improvement by addressing challenges through collaborative problem-solving. I encourage individual reflection logs which could further enrich learning and project outcomes.

**Comment by JunyiLi0804** [comment]
Yes, it makes sense, but as a team we need to focus our reflection and attention more on the team's whole shortcomings rather than individual reflection logs, but if time is not limited, it will be taken our consideration
